test/pages/section.page.js

```diff
@@ -34,13 +37,6 @@ class Section {
         commonActions.pauseInSeconds(3);
         let finalSelector = commonActions.getSelector(section);
         commonActions.clickWebElement(finalSelector);
-        if (isClassic) {
-            commonActions.closePopMessage(this.lightningMessage.message, this.lightningMessage.closeButton);
```

Comment:
Are you sure? this line is to close Spam message after choose any section.

Reply:
sure it is not neccesary.I tried it